movement/watch_faces/clock/simple_clock_face.c

```diff
@@ -134,16 +142,31 @@ bool simple_clock_face_loop(movement_event_t event, movement_settings_t *setting
             else watch_clear_indicator(WATCH_INDICATOR_BELL);
             break;
         case EVENT_BACKGROUND_TASK:
+            date_time = watch_rtc_get_date_time();
             // uncomment this line to snap back to the clock face when the hour signal sounds:
             // movement_move_to_face(state->watch_face_index);
             if (watch_is_buzzer_or_led_enabled()) {
                 // if we are in the foreground, we can just beep.
+#if SIGNAL_SUB_SIMPLE_TUNE
+                if (date_time.unit.minute == 0)
+                    movement_play_signal();
+                else
+                    watch_buzzer_play_sequence(signal_sub_tune, NULL);
+#else
                 movement_play_signal();
+#endif
             } else {
                 // if we were in the background, we need to enable the buzzer peripheral first,
                 watch_enable_buzzer();
                 // beep quickly (this call blocks for 275 ms),
+#if SIGNAL_SUB_SIMPLE_TUNE
+                if (date_time.unit.minute == 0)
+                    movement_play_signal();
+                else
+                    watch_buzzer_play_sequence(signal_sub_tune, NULL);
+#else
                 movement_play_signal();
+#endif
                 // and then turn the buzzer peripheral off again.
                 watch_disable_buzzer();
```

Comment:
This will be problematic in conjunction with `watch_buzzer_play_sequence()`. See [this discussion.](https://github.com/joeycastillo/Sensor-Watch/issues/275#issuecomment-1730809624)